lib/ansible/utils/module_docs_fragments/cloudstack.py

```diff
@@ -20,12 +20,12 @@ class ModuleDocFragment(object):
   api_http_method:
     description:
       - HTTP method used.
-    default: get
+      - Fallback is C(get) if not specified.
```

Comment:
While you're fixing these it would be good to add information about the other sources of fallback settings (environment vars and configuration).